### ISSUE TITLE
`cytoolz` version conflict after update to Populus 2.2.0

### ISSUE THREAD

**Issue by veox** [ethereum/populus#416]
* OS: Arch Linux (last full update 2017-12-13)
* Python: 3.6.3
* `pip`: 9.0.1
* Populus: 2.2.0

### What's wrong?

Upgrading to Populus 2.2.0 (from 2.1.0) breaks package dependencies.

Post-upgrade in an existing Solidity `Greeter` project, running `pytest` fails with:

``` python
Traceback (most recent call last):
  File "/mnt/data/veox/src/populus-greeter/.virtualenv/greeter/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 510, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/mnt/data/veox/src/populus-greeter/.virtualenv/greeter/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2404, in load
    self.require(*args, **kwargs)
  File "/mnt/data/veox/src/populus-greeter/.virtualenv/greeter/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2427, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/mnt/data/veox/src/populus-greeter/.virtualenv/greeter/lib/python3.6/site-packages/pkg_resources/__init__.py", line 875, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (cytoolz 0.9.0 (/mnt/data/veox/src/populus-greeter/.virtualenv/greeter/lib/python3.6/site-packages), Requirement.parse('cytoolz==0.8.2'), {'eth-tester'})

During handling of the above exception, another exception occurred:
<yada, more of the same>
_pytest.vendored_packages.pluggy.PluginValidationError: Plugin 'ethereum' could not be loaded: (cytoolz 0.9.0 (/mnt/data/veox/src/populus-greeter/.virtualenv/greeter/lib/python3.6/site-packages), Requirement.parse('cytoolz==0.8.2'), {'eth-tester'})!
```

That is, `eth-tester` [specifies](https://github.com/ethereum/eth-tester/blob/6f4c67c400aea54ad2d012ec92ec9f8e8a729a45/setup.py#L20) a `cytoolz==0.8.2` dependency, but `0.9.0` gets installed instead.

This is strange: I'd expect `pip` to detect the fact, and either install `cytoolz==0.8.2`, or, if something else requires `cytoolz>=0.9.0`, then warn the user.

### Workaround

After installing Populus, roll back the `cytoolz` dependency:

``` sh
pip install cytoolz==0.8.2
```

After this, issue #414 should be experienced on running `pytest`.

#### Cute Animal Picture

![](http://effectivewildlifesolutions.com/wp-content/uploads/2015/02/Curious_Raccoon2.jpg)

**Comment by pipermerriam** [comment]
Working my way up the dependency tree to fix this.  https://github.com/ethereum/eth-tester/pull/36

**Comment by voith** [comment]
@veox This was resolved in #474. Also, the tester chain now uses `eth-tester`. 
closing for now. Please re-open if problem still persists.
